backend/models/listing.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const ListingSchema = new mongoose.Schema({
+    listerID: {type: mongoose.Schema.Types.ObjectId, required: true},
+    name: {type: String, required: true, minlength: 5, maxlength: 30},
+    description: {type: String},
+    rental_price: {type: Number, required: true},
+    date: {type: Date},
```

Comment:
Should we make this required? (Especially if it is automatically assigned via our backend)